common/config/beachball/changelog-config.ts

```diff
@@ -7,7 +7,12 @@ export const config: BeachballConfig = {
     customRenderers: {
       renderHeader,
       renderEntry
-    }
+    },
+    groups: [{
```

Comment:
small change that handily rolls up all packages changelogs into the meta package changelog